app/services/user_service/update_card_locking.rb

```diff
@@ -20,7 +20,9 @@ def run
         TwilioMessageService::Send.new(@user, message).run!
       end
 
-      @user.update!(cards_locked: cards_should_lock)
+      User.with_2fa_requirement_disabled do
+        @user.update!(cards_locked: cards_should_lock)
+      end
```

Comment:
Can you please send the user ID of the user without 2FA enabled to me?

Reply:
lol - look who it was.

<img width="214" height="164" alt="Screenshot 2025-08-13 at 3 26 25 PM" src="https://github.com/user-attachments/assets/47fc9ce3-5cae-419e-9a7c-e144094693bf" />